services/accounts/accounts_test.go

```diff
@@ -5,39 +5,44 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/golang/mock/gomock"
 	empty "github.com/golang/protobuf/ptypes/empty"
 	"github.com/google/go-cmp/cmp"
-	"github.com/stretchr/testify/mock"
 
 	accountspb "github.com/openbank/gunk/gunk/v1/accounts"
 	typespb "github.com/openbank/gunk/gunk/v1/types"
-	"github.com/openbank/openbank/storage"
+	"github.com/openbank/openbank/storage/mock_storage"
 )
 
 func TestHandler_GetAccount(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	m := mock_storage.NewMockAccountStore(ctrl)
```

Comment:
Just `NewAccountStore` is enough. We do have `mock_storage` prefix already.

Reply:
`mock_storage.go` is a generated file by `golang/mock` gen tool :)